crates/sui-framework/build.rs

```diff
@@ -92,14 +118,23 @@ fn build_packages(sui_system_path: PathBuf, sui_framework_path: PathBuf, out_dir
 }
 
 fn build_packages_with_move_config(
+    deepbook_path: PathBuf,
     sui_system_path: PathBuf,
     sui_framework_path: PathBuf,
     out_dir: PathBuf,
+    deepbook_dir: &str,
     system_dir: &str,
     framework_dir: &str,
     stdlib_dir: &str,
     config: MoveBuildConfig,
 ) {
+    let deepbook_pkg = BuildConfig {
```

Comment:
You may want to do what Ashok did in #10593 and build `framework_pkg` before deepbook to avoid unwanted doc changes due to residual build artifact.